src/main/java/com/google/cloud/spanner/pgadapter/statements/DdlExecutor.java

```diff
@@ -202,8 +202,8 @@ ImmutableList<Statement> getDependentStatements(Statement statement) {
     if (!parser.eatKeyword("drop")) {
       return defaultResult;
     }
-    parser.eatKeyword("if", "exists");
     if (parser.eatKeyword("table")) {
+      parser.eatKeyword("if", "exists");
```

Comment:
What if the table doesn't exist? Will it just return an error? Can we add a test case for that?

Reply:
Added